app/models/user.rb

```diff
@@ -87,6 +87,12 @@ def watching?(task)
     self.tasks.where(id: task.id).exists?
   end
 
+  def search_tasks(search)
+    self.projects.each do |project|
+      return project.tasks.where("title ~* ?", "\\m#{search}").map
```

Comment:
where is some limits in results? If I type "a" I will get 100000 results from DB. return only 10 per type/search